Core/Battler.gd

```diff
@@ -68,6 +68,9 @@ func _on_enemy_start_turn() -> void:
 		var success: bool = _on_attack(enemy.get_behavior_component().attack, enemy, PlayerManager.player)
 		assert(success == true, "Enemy failed to attack.")
 	
+	# TODO: temporary delay so we can see the draw pile and discard pile working
+	await get_tree().create_timer(1.0).timeout
```

Comment:
timer time should be a variable so it's more easily accessible

Reply:
Yeah fair. This is gonna get removed whenever we add enemy attack animations, but I can add the variable for now